src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -204,12 +205,16 @@ public function normalize($data, string $format = null, array $context = [])
      */
     public function denormalize($data, string $type, string $format = null, array $context = [])
     {
+        if (isset($context[DenormalizerInterface::COLLECT_DENORMALIZATION_ERRORS], $context['not_normalizable_value_exceptions'])) {
```

Comment:
`if (isset($context['not_normalizable_value_exceptions'])) {` ?

Reply:
This is not possible, since the serializer is called recursively